DotNetHowToAutocomplete/DotNetHowToAutocomplete/Views/Home/Index.cshtml

```diff
@@ -126,8 +128,8 @@
     <div class="example">
         <h2 class="exampleTitle">Both Suggestions and Autocomplete</h2>
         <div class="exampleText">
-            Autocomplete provides terms that are in the index and help the user complete what they are currently typing.  This helps to user write their search terms as they get recommendation for words and phrases that appear in the index.  This can also be used to provide recommendations for search terms that have been previously been used by other users.
-            You can use the <a href="TBD">AUTOCOMPLETE</a> method on the Azure Search REST API to retrieve words that complete a given search term.<br /><br />
+            Autocomplete and suggestions can work together to enhance the user's search experience.  The search box completes the word current typed using the first autocomplete recommendation returned and suggestions appear in a dropdown below the search box.
+            This example uses both the Suggest and Autocomplete method in the Azure Search .Net SDK.<br /><br />
```

Comment:
I noticed that the `Both` method was removed from `HomeController`, but this section of the UI is still here. I assume that means you make two separate calls to the controller, and stitch the results together on the client. Is that correct?

Reply:
Correct.